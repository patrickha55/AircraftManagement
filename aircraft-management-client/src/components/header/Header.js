import { NavLink } from "react-router-dom"
import React from 'react';

const Header = () => {
    return (
        <header>
            <div className="px-3 py-2 bg-dark text-white">
                <div className="container">
                    <div className="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
                        <ul className="nav col-12 col-lg-auto my-2 justify-content-center my-md-0 text-small">
                            <NavLink to="/aircraft" className="nav-link text-white">
                                Aircraft
                            </NavLink>
                            <NavLink to="/Roles" className="nav-link text-white">
                                Roles
                            </NavLink>
                        </ul>
                    </div>
                </div>
            </div>
        </header>
    )
}

export default Header