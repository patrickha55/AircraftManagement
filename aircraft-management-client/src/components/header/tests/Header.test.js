import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Header from '../Header';

test("Header should renders", () => {
    const header = shallow(<Header />);

    expect(toJson(header)).toMatchSnapshot();
});