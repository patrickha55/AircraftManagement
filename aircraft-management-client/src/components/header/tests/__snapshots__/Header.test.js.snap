// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders 1`] = `
<header>
  <div
    className="px-3 py-2 bg-dark text-white"
  >
    <div
      className="container"
    >
      <div
        className="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start"
      >
        <ul
          className="nav col-12 col-lg-auto my-2 justify-content-center my-md-0 text-small"
        >
          <NavLink
            className="nav-link text-white"
            to="/aircraft"
          >
            Aircraft
          </NavLink>
          <NavLink
            className="nav-link text-white"
            to="/Roles"
          >
            Roles
          </NavLink>
        </ul>
      </div>
    </div>
  </div>
</header>
`;
